index.js

```diff
@@ -725,6 +725,198 @@ PersonaClient.prototype._removeTokenFromCache = function (id, secret, callback)
 };
 
 /**
+ * Get scope information for a user
+ * @param guid
+ * @param token
+ * @param callback
+ */
+PersonaClient.prototype.getScopesForUser = function(guid, token, callback) {
+    try {
+        _.map([guid, token], function (arg) {
+            if (!_.isString(arg)) {
+                throw "guid and token are required strings";
```

Comment:
Apologies, turns out I misunderstood the map function. You can't return early from it so try/catch is fine for this approach now.